content/en/monitors/notify/notification_rules.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Notification Rules
+further_reading:
+- link: "/monitors/notify/"
+  tag: "Documentation"
+  text: "Configure monitor notifications"
+- link: "/monitors/settings/"
+  tag: "Documentation"
+  text: "Monitor Settings"
+---
+
+## Overview
+
+Monitor notification rules are predefined sets of conditions that automate the process of alerting your team based on predefined conditions and tags. Instead of configuring notification recipients and routing for every monitor individually, Notification rules allow you to define the notification logic and recipients in one place and automatically route all monitor events with matching tags to that list of handles.
```

Comment:
```suggestion
Monitor notification rules are predefined sets of conditions that automate the process of alerting your team based on predefined conditions and tags. Instead of individually configuring notification recipients and routing for every monitor, notification rules allow you to define the notification logic and recipients in one place and automatically route all monitor events with matching tags to that list of handles.
```